facilitator/solutions/tower_install.md

```diff
@@ -1,29 +1,30 @@
 # Workshop: Installing Tower
 
-Here students are tasked with installing and activating a new single integrated instance of Ansible Tower. The assignment will show students how they can run an instance of Tower themselves for testing, demos and their own proof-of-concepts. 
+Here students are tasked with installing and activating a new single integrated instance of Ansible Tower. The assignment will show students how they can run an instance of Tower themselves for testing, demos and their own proof-of-concepts.
 
-#### Tips
+## Tips
 
 If you are working in an environment that does not have access repos such as EPEL, you can opt to use the [Ansible Tower bundle installer](http://releases.ansible.com/ansible-tower/setup-bundle/ansible-tower-setup-bundle-latest.el7.tar.gz) instead. This bundle includes all of the RPMs from these repositories. It does not include **all** of the RPMs required.
 
 Once downloaded, configured and the `setup.sh` is started, is will take a few minutes to install all of the Ansible Tower dependencies. Students can take a short break while that runs.
 
 ### Solution
 
-1. Retrieve the latest version of Tower from the Ansible Tower releases archive.  
-2. Unarchive the retrieved Ansible Tower archive.
-    * i.e. `tar -vxzf towerlatest` 
-3. Change directories to the one created by the expanded Tower software archive.
-4. Edit the `inventory` file with passwords for admin, message queue and the postgres database will use while setting up. These are represented by these inventory variables:
-    * `admin_password` 
+1. Retreive the latest version of Tower from the Ansible Tower releases archive.
+1. Unarchive the retreived Ansible Tower archive.
```

Comment:
typo retrieved

Reply:
We received a PR that was merged that fixes a lot of the mistakes which is probably why this PR that @bbaassssiiee submitted has conflicts and needs to be resolved.